advertising/transaction-message/reference.md

```diff
@@ -112,13 +115,15 @@ Defines a type of room.
 |Element|Description|Children
 |-|-|-
 |RoomID|Required.<br />Data type is string.<br /><br />An ID that uniquely identifies the room. If your \<Result> element contains \<RoomBundle>, set the bundles' \<RoomID> element to this ID, as appropriate. You may also use this ID to set the `Result` object's \<RoomID> element, if you don't use room bundles but want to identify a type of room.|None
+|Room Size| Optional. Size of the room in square meter or square feet. | [Text Type](#text-type)
+|Bed arrangements| Optional. The arrangement of beds in this room. (for example: 1 Extra-large double bed(s) (Super-king size))| [Text Type](#text-type)
```

Comment:
We can describe the Bed Arrangement as vector of <BedTypes> each containing BedType and Count.
<BedArrangement>
<BedType></BedType>
<BedCount></BedCount>
<BedType></BedType>
<BedCount></BedCount>
</BedArrangement>